app/src/main/java/com/it3048/mobiledeviceproject/AddMeetingForm.kt

```diff
@@ -30,20 +30,20 @@ class AddMeetingForm: DialogFragment() {
 
 
         rootView.submitButton.setOnClickListener{
-            //this is where the inputs get saved to state
-//            var meetingTitle = rootView.editTextMeetingTitle.text.toString()
-//            var meetingColor = rootView.meetingColor.selectedItem.toString()
-//            var meetingDate = rootView.meetingDate.text.toString()
-//            var meetingDescription = rootView.editTextDescripton.text.toString()
-//            var meetingLocLink = rootView.editTextLocLink.text.toString()
-//
-//            Toast.makeText(context, "submitted", Toast.LENGTH_LONG).show()
-//            Log.i(meetingTitle, "meetingTitle")
-//            Log.i(meetingColor, "meetingColor")
-//            Log.i(meetingDate, "meetingDate")
-//            Log.i(meetingDescription, "meetingDescription")
-//            Log.i(meetingLocLink, "meetingLocLink")
+            /**this is where the inputs get saved to state
+            var meetingTitle = rootView.editTextMeetingTitle.text.toString()
+            var meetingColor = rootView.meetingColor.selectedItem.toString()
+            var meetingDate = rootView.meetingDate.text.toString()
+            var meetingDescription = rootView.editTextDescripton.text.toString()
+            var meetingLocLink = rootView.editTextLocLink.text.toString()
 
+            Toast.makeText(context, "submitted", Toast.LENGTH_LONG).show()
+            Log.i(meetingTitle, "meetingTitle")
+            Log.i(meetingColor, "meetingColor")
+            Log.i(meetingDate, "meetingDate")
+            Log.i(meetingDescription, "meetingDescription")
+            Log.i(meetingLocLink, "meetingLocLink")
+ */
```

Comment:
Committing commented out code is considered bad practice in version control, because version control offers better alternatives:
1) Use a feature branch for experimental features.
2) Delete what is not needed.  You can always compare with history to see what used to be there.
Commented out code is considered clutter in a version control system.